wowchemy/layouts/partials/breadcrumb.html

```diff
@@ -0,0 +1,10 @@
+{{ if not .IsHome }}
+  <nav aria-label="breadcrumb">
+    <ol class="breadcrumb">
+      {{ partial "breadcrumb_helper" . }}
+      <li class="breadcrumb-item active" aria-current="page">
+        {{- .Title | emojify -}}
```

Comment:
For consistency, please use the same link naming logic that already exists in `book_menu.html`, i.e. `{{ safeHTML $icon }}{{ .LinkTitle | default .Title }}`

If you wish to also parse titles with `emojify`, please add it to the link naming instances in `book_menu.html`.

Reply:
`$icon` is set in `book_menu` as you say, which is a lower level partial than `book_layout`. I can probably dig out the html from `$current_page` if you think that's important.